pyproject.toml

```diff
@@ -39,7 +39,7 @@ pydantic = ">=1.8,<2"
 dacite = ">=1.6,<2"
 requests = ">=2.20,<2.30" # TODO: revert upper bound when urllib3 situation sorts out: https://github.com/psf/requests/issues/6432
 requests-toolbelt = "*"
-importlib-metadata = { version = "<5.0", python = "<3.8" }
+importlib-metadata = { version = "*", python = "<3.8" }
```

Comment:
nit: latest version `6.8.0` requires `python>=3.8`. `6.7.0` is good for `python >=3.7`

Reply:
Pip should be smart enough to figure out the max version to install given the current Python version. As far as I am aware, we don't need to pin an upper bound here directly.